### PR TITLE
[DM-30697] Update token documentation for new UI

### PR DESCRIPTION
**Pull request by rra**
Document how to create a new user token through the token UI rather
than copying the notebook token.  Make the specific instructions
using TOPCAT conditional on whether the environment deploys a TAP
service and only provide generic token creation documentation
otherwise.

### CODE REVIEW

File: src/environment/tokens.rst [hunk 1]
```diff
@@ -5,91 +5,122 @@ Science Platform Tokens
...
 An example is the TAP service provided by the API Aspect.
 When interacting with authenticated services for within the Notebook Aspect you typically doesn't need to interact with the access tokens since the various clients are able to pick up the token from your user environment.
-This access token can also be used outside the Notebook Aspect to access services from a local machine (for example, to access the TAP service from `TOPCAT`_).
+If you want to access services from a local machine (for example, to access the TAP service from `TOPCAT`_), you will need to create an access token for that purpose.
 
-This page will give you an introduction to the access token, where to find it, how to inspect it, and how to use it in an environment outside the Notebook Aspect.
-Note that the access token has a finite lifetime and is refreshed on each login to the Notebook Aspect.
+This page will give you an introduction to the access token, where to create onefor use from a local machine, and how to use it in an environment outside the Notebook Aspect.
```
Comment (jonathansick) [inline src/environment/tokens.rst]:
```suggestion
This page will give you an introduction to the access token, where to create one for use from a local machine, and how to use it in an environment outside the Notebook Aspect.
```

**Review by jonathansick — APPROVED** [review]
The updates to `rsp-environments.yaml` adn the linter look great. I also built the `summit` docs locally to see how the TAP-less docs look and they're great.

Just one type.

A question, I hadn't noticed the `x-oauth-basic` username thing before. Do all VO services go by username+password and require the `x-oauth-basic` username to work? I'll add that to the squareone version of this doc.

File: src/environment/tokens.rst [hunk 2]
```diff
@@ -5,91 +5,122 @@ Science Platform Tokens
...
 Executing the two cells in the notebook will provide information about token creation time and lifetime among other things.
 You will note from the above notebook that the token is also provided as the value of the environment variable ``ACCESS_TOKEN``, but the file version should be considered primary.
 
 Using a token outside the Science Platform
 ==========================================
 
-Following is a recipe for an example workflow that uses the access token taken from a JupyterLab session to access the Science Platform TAP service from a local computer using TOPCAT.
-You can apply similar steps to other use cases.
+If you want to use an access token from your local system, you can create a new one.
+You can restrict what that token is able to do to only the services that you need to access, give the token a name, and configure its expiration (or set it to not expire).
```
Comment (SimonKrughoff) [inline src/environment/tokens.rst]: Maybe use a bullet list to make it more readable?
---
At creation time you can:

- restrict access to only the services you need
- give the token a name
- configure the expiration including setting it to not expire

**Comment by rra** [comment]
So, the real answer is that people shouldn't use Basic Auth with tokens. They should use the token as a bearer token, sending `Authorization: bearer <token>` in the headers.

If you have to use basic auth because the program doesn't support proper token authentication (such as with TOPCAT so far as I can tell), Gafaelfawr currently supports three modes: `x-oauth-basic` as the username and the token as the password, the token as the username and `x-oauth-basic` as the password, or the token as the username and no password. Only one of those puts the token in the password field, which seems more appropriate since it's a secret value and thus shouldn't be too readily available in UIs, etc. So I changed the documentation to use that. (But the old way it was doing things is still supported.)

**Review by SimonKrughoff — APPROVED** [review]
Just one formatting suggestion

**Pull request merged**